src/nifgen/metadata/config.py

```diff
@@ -0,0 +1,26 @@
+config = {
+    'metadata_version': '1.0',
+    'module_name': 'nifgen',
+    'module_version': '0.4.0.dev0',
+    'c_function_prefix': 'niFgen_',
+    'driver_name': 'NI-FGEN',
+    'session_class_description': 'An NI-FGEN session to a National Instruments Function Generator.',
```

Comment:
Function Generator -> Signal Generator

"function generators" is a subset of "signal generators", the other type of "signal generator" we sell being an "arbitrary waveform generator".

Reply:
Fixed